include/date/tz.h

```diff
@@ -838,7 +838,7 @@ class time_zone
     DATE_API void init() const;
     DATE_API void init_impl();
     DATE_API sys_info
-        load_sys_info(std::vector<detail::transition>::const_iterator i) const;
+        load_sys_info(std::vector<detail::transition>::const_iterator i, sys_seconds tp) const;
```

Comment:
With the change above (Including posix timezones) we also can no longer uniquely identify a timezone by its transition information from the TZIF file because the posix timezone might be in effect. 

Passing the timepoint in was the simplest solution I could find